packages/alfa-css/src/syntax/token.ts

```diff
@@ -1346,16 +1346,14 @@ export namespace Token {
   export const parseCloseComment = parseToken(isCloseComment);
 }
 
-function parseToken<T extends Token>(
-  refinement: Refinement<Token, T>
-): Parser<Slice<Token>, T, string> {
-  return (input) =>
+function parseFirst(): Parser<Slice<Token>, Token, string> {
+  return (input: Slice<Token>) =>
     input
       .first()
-      .map((token) =>
-        refinement(token)
-          ? Ok.of<[Slice<Token>, T]>([input.rest(), token])
-          : Err.of("Mismatching token")
-      )
+      .map((token) => Ok.of<[Slice<Token>, Token]>([input.rest(), token]))
       .getOr(Err.of("No token left"));
 }
+
+function parseToken<T extends Token>(refinement: Refinement<Token, T>) {
+  return parseIf(refinement, parseFirst(), (_) => "Mismatching token");
```

Comment:
```suggestion

function parseToken<T extends Token>(refinement: Refinement<Token, T>) {
  return parseIf(refinement, parseFirst, () => "Mismatching token");
```
(`_` doesn't have a special meaning in ES, so if we don't need more parameters, we shouldn't create one)